openstack/bms/v2/tags/doc.go

```diff
@@ -1,25 +0,0 @@
-/*
```

Comment:
Why you delete all docs?

Reply:
We still need to discuss about the documentation: should we have a dedicated documentation page like TF, or should we say put all the documentation in doc.go?
But the docs are removed here because the descriptions are outdated and we should have a series of work to write and revise the docs.

Reply:
Added statement as an issue: https://github.com/opentelekomcloud/gophertelekomcloud/issues/406. Maybe need to be moved to discussions